sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -43,13 +41,26 @@ class ModelFileType(enum.Enum):
   JOBLIB = 2
 
 
-class SklearnModelHandler(ModelHandler[Union[numpy.ndarray, pandas.DataFrame],
-                                       PredictionResult,
-                                       BaseEstimator]):
-  """ Implementation of the ModelHandler interface for scikit-learn.
+def load_model(model_uri, file_type):
```

Comment:
Should this be private with a leading underscore? (Another idea is to put it in a common base class, but there are pros and cons to that so I'm fine with this.)

Reply:
yep it should be private. I don't think it should go in the base class, because it is sklearn specific and will not be used outside this module.

Reply:
I was thinking a new abstract SkLearn base class, but that's probably not worth its weight.